### PR TITLE
Early Game Check

### PR DESCRIPTION
**Pull request by vtange**

### CODE REVIEW

**Comment by vtange** [comment]
Good To Merge

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -9579,14 +9579,14 @@ QUEST_LV_0100_20151001_009574	{memo X}You have to save the mysterious girl while
 QUEST_LV_0100_20151001_009575	{memo X}You've obtained the stone!
 QUEST_LV_0100_20151001_009576	You couldn't obtain anything!
 QUEST_LV_0100_20151016_009577	
-QUEST_LV_0100_20151016_009578	
-QUEST_LV_0100_20151016_009579	
+QUEST_LV_0100_20151016_009578	$You mean you want to enter the Crystal Mines to find the evidence of Salvation? {nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Crystal Mine

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -9579,14 +9579,14 @@ QUEST_LV_0100_20151001_009574	{memo X}You have to save the mysterious girl while
...
+QUEST_LV_0100_20151016_009579	$So we are not in the situation to spare our troops for guarding Revelators while they check the evidence of salvation. {nl}But we also can't go against the order of the Bishop and Knight Commander either. {nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:9583]: salvation isn't capitalized here. To capitalize it or not?

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -9579,14 +9579,14 @@ QUEST_LV_0100_20151001_009574	{memo X}You have to save the mysterious girl while
...
 QUEST_LV_0100_20151016_009580	
 QUEST_LV_0100_20151016_009581	
 QUEST_LV_0100_20151016_009582	
 QUEST_LV_0100_20151016_009583	
 QUEST_LV_0100_20151016_009584	
-QUEST_LV_0100_20151016_009585	
+QUEST_LV_0100_20151016_009585	$Anyway, the bishop himself also had a dream while praying in closure. {nl}In his dream, the Goddess told him the evidence of salvation will be found in the Crystal Mine. {nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: To capitalize Bishop or not because in line 9583 it is.

File: QUEST_LV_0100.tsv (hunk 2 above)
Comment (vtange) [inline QUEST_LV_0100.tsv:9583]: i would if we're capitlizing all of Evidence of Salvation, as if it's an item

File: QUEST_LV_0100.tsv (hunk 3 above)
Comment (vtange) [inline QUEST_LV_0100.tsv]: probably capt. since there seems to be only one Bishop of klaipeda

Comment (Sunaries) [inline QUEST_LV_0100.tsv]: There are several bishops. For instance, Maven is considered a bishop, although an archbishop.
In Orsha there is Bishop Urbonas.

But I guess this 'bishop' is referring to Aurelius.

File: QUEST_LV_0100.tsv (hunk 2 above)
Comment (Sunaries) [inline QUEST_LV_0100.tsv:9583]: We should leave it as 'evidence of salvation' - Checked ITEM real quick, there's only "Relic of Salvation" used in the Aurelius story.

**Comment by vtange** [comment]
good to merge

**Comment by Sunaries** [comment]
Good. :+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**